<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QMainWindow" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditorWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,2,0">
      <item>
       <widget class="QComboBox" name="currentObjectComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="commentPlainTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>(No comment for current object)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ObjectTreeWidget" name="objectListTreeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="DropTestLabel" name="dropTestLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>Drop Test</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="placeholderWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="placeholderLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string notr="true">placeholderLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_Object">
    <property name="title">
     <string>File/Object</string>
    </property>
    <widget class="QMenu" name="menuCapture">
     <property name="title">
      <string>Capture...</string>
     </property>
     <addaction name="actionCaptureClipboard"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionChangeDirectory"/>
    <addaction name="menuCapture"/>
   </widget>
   <widget class="QMenu" name="menuMisc">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionOpenLog"/>
   </widget>
   <widget class="QMenu" name="menuCrash">
    <property name="title">
     <string>Crash</string>
    </property>
    <addaction name="actionQFatal"/>
    <addaction name="actionSegFault"/>
   </widget>
   <addaction name="menuFile_Object"/>
   <addaction name="menuCrash"/>
   <addaction name="menuMisc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionChangeDirectory">
   <property name="text">
    <string>Change Directory...</string>
   </property>
  </action>
  <action name="actionCaptureClipboard">
   <property name="text">
    <string>Clipboard</string>
   </property>
  </action>
  <action name="actionOpenLog">
   <property name="text">
    <string>Open log file</string>
   </property>
  </action>
  <action name="actionQFatal">
   <property name="text">
    <string>qFatal()</string>
   </property>
  </action>
  <action name="actionSegFault">
   <property name="text">
    <string>Segmentation fault</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObjectTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>src/gui/ObjectTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>DropTestLabel</class>
   <extends>QLabel</extends>
   <header>src/gui/DropTestLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
